tilings/tiling.py

```diff
@@ -64,16 +67,31 @@ class Tiling(CombinatorialClass):
     cells and the active cells.
     """
 
+    func_calls = func_calls
+    func_times = func_times
+
+    # @cssmethodtimer("Tiling.__init__")
     def __init__(
         self,
         obstructions: Iterable[GriddedPerm] = tuple(),
         requirements: Iterable[Iterable[GriddedPerm]] = tuple(),
         assumptions: Iterable[TrackingAssumption] = tuple(),
-        remove_empty: bool = True,
+        remove_empty_rows_and_cols: bool = True,
         derive_empty: bool = True,
-        minimize: bool = True,
+        simplify: bool = True,
         sorted_input: bool = False,
+        already_minimized_obs: bool = False,
     ) -> None:
+        """
+        - if not sorted_input, input will be sorted
+        - if minimize, will be minimized
```

Comment:
This doctrstring does not match the actual arguments of the method

Reply:
Fixed.